src/content/docs/mlops/integrations/openai-integration.mdx

```diff
@@ -4,58 +4,65 @@ metaDescription: "A lightweight tool to monitor your OpenAI workload."
 freshnessValidatedDate: never
 ---
 
-import modelperformancemonitoringOpenAIDashboard from 'images/model-performance-monitoring_screenshot-crop_openai-dashboard.webp'
+import modelperformancemonitoringOpenAIDashboard from 'images/model-performance-monitoring_screenshot-crop_openai-dashboard-1.webp'
 
-With the GPT series application integration, users have the ability to seamlessly monitor OpenAI completion queries and simultaneously log useful statistics in a New Relic customizable dashboard.
+With New Relic's GPT series application integration, you have the ability to seamlessly monitor all OpenAI queries. By adding just two lines of code, your team can gain access to key performance metrics such as cost, response time, rate limits, error rates, chat history, and sample inputs and outputs. Our GPT integration provides real-time metrics to help businesses optimize usage, reduce costs, and achieve better results.
 
-By adding just two lines of code, users can gain access to key performance metrics such as cost, response time, and sample inputs/outputs. The dashboard also allows users to track total requests, average token/requests, and model names.
-
-With the fully customizable dashboard of New Relic, users can add additional metrics based on their specific business requirements and needs.
-
-Overall, New Relic's GPT integration provides real-time metrics to help businesses optimize usage, reduce costs, and achieve better results.
+This guide will show you how to integrate you OpenAI GPT applications so you can start monitoring your usage.
 
+Once you instrument find OpenAI observability data under **Model Performance** >  **Application**.
 
 <img
```

Comment:
I'd move this screenshot under your intro paragraph, then as a second paragraph add something like:

> This guide will show you how to integrate you OpenAI GPT applications so you can start monitoring your usage. When you're ready, find your OpenAI data under Model Performance > Application.

Reply:
So:

Intro paragraph
Screenshot
This snippet of text

Reply:
idk how this would look though. i'd test it out and choose whatever looks less janky